deploy/helm/airflow-operator/crds/crds.yaml

```diff
@@ -537,38 +537,42 @@ spec:
                       items:
                         properties:
                           branch:
-                            description: The branch to clone. Defaults to `main`. Since git-sync v4.x.x this field is mapped to the flag `--ref`.
-                            nullable: true
+                            default: main
+                            description: |-
+                              The branch to clone. Defaults to `main`.
+
+                              Since git-sync v4.x.x this field is mapped to the flag `--ref`.
                             type: string
                           credentialsSecret:
                             description: 'The name of the Secret used to access the repository if it is not public. This should include two fields: `user` and `password`. The `password` field can either be an actual password (not recommended) or a GitHub token, as described [here](https://github.com/kubernetes/git-sync/tree/v4.2.4?tab=readme-ov-file#manual).'
                             nullable: true
                             type: string
                           depth:
+                            default: 1
                             description: The depth of syncing i.e. the number of commits to clone; defaults to 1.
-                            format: uint8
+                            format: uint32
                             minimum: 0.0
-                            nullable: true
                             type: integer
                           gitFolder:
-                            description: The location of the DAG folder, relative to the synced repository root.
-                            nullable: true
+                            default: ''
+                            description: The location of the DAG folder, relative to the synced repository root. An empty string ("") corresponds to the root folder.
```

Comment:
This would be better as `/`. An empty string carries too much ambiguity.
If that is not possible, please ignore.

Reply:
I agree and was thinking the same!
However, this would require something more advanced than string concatenation. I though to not further bloat this PR, however would you preferring adding it?

Reply:
Please have a look at a657de0